packages/strapi-plugin-i18n/admin/src/components/ModalEdit/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Modal, ModalHeader, ModalSection, ModalFooter } from 'strapi-helper-plugin';
+import { useIntl } from 'react-intl';
+import { Button, Padded } from '@buffetjs/core';
+import PropTypes from 'prop-types';
+import { Row } from 'reactstrap';
+
+import { getTrad } from '../../utils';
+
+const ModalEdit = ({ isLoading, isOpen, onCancel, onClosed, onClick, onOpened, onToggle }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Modal isOpen={isOpen} onOpened={onOpened} onToggle={onToggle} onClosed={onClosed}>
+      <ModalHeader headerBreadcrumbs={['Edit locale']} />
+      <ModalSection>
+        <div>
+          <Padded top size="md">
+            <Row>Put the form here</Row>
```

Comment:
cc @soupette , not sure how to put the base / advanced form here 🤔

Reply:
An approach would be to create a stepper, to control the form you will be displaying depending on the 'tab';

**`stepper.js`**
```
const stepper = {
 // Layout for the form
  base: [
      [
       { name: 'key', type: 'text', label: { id: 'translation_id'} }
      ]
   ],
  advanced: [],
}
```

Regarding the components needed for the 'tabs' we have a [component](https://github.com/strapi/strapi/blob/0771b659d5f4541600360402a7bee29139ae3a2a/packages/strapi-plugin-content-type-builder/admin/src/containers/FormModal/index.js#L1107-L1129) in the `content-type-builder` that could work but we'll have to move it into the `strapi-helper-plugin` to avoid code duplication.

What do you think?